sensorhub-core/src/main/java/org/sensorhub/impl/system/DefaultSystemRegistry.java

```diff
@@ -282,7 +282,10 @@ public synchronized void registerDatabase(String uid, IObsSystemDatabase db)
             systemStateDb.getDataStreamStore().removeEntries(dsFilter);
             systemStateDb.getCommandStreamStore().removeEntries(csFilter);
             var count = systemStateDb.getSystemDescStore().removeEntries(procFilter);
-            
+
+            // Replace driver's transaction handler so that new IObsSystemDatabase handles driver
+            register(getDriverHandler(uid).driver);
```

Comment:
Would this be an appropriate replacement? Of course moved up before the removes.
```java
systemStateDb.getSystemDescStore().selectEntries(procFilter).forEach(desc ->
register(getDriverHandler(desc.getValue().getUniqueIdentifier()).driver));
```

Reply:
I couldn't use the provided procFilter because it has `includeMembers()` set to false by default. So I included it in another filter and it is working properly.